### PR TITLE
Fix #83: Pull latest charts, show correct date

### LINKED ISSUE #83: Show the most recent data for all charts and show an accurate date in the heading
Now that the `/metric/#/` endpoint supports `?date` query strings, we should try to retrieve the latest data for all charts. The endpoint also returns the date of the nearest matching dataset, which we should display in the header.

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: distributionviewer/core/static/js/app/components/views/chart.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ export class Chart extends React.Component {
   }
 
   componentWillMount() {
-    axios.get(`${metricApi.endpoints.GET_METRIC}${this.props.chartId}/`).then(response => {
+    const todayISO = new Date().toISOString().substring(0, 10);
+    axios.get(`${metricApi.endpoints.GET_METRIC}${this.props.chartId}/?date=${todayISO}`).then(response => {
```
Comment (robhudson) [inline distributionviewer/core/static/js/app/components/views/chart.js]: Technically this isn't required since the same thing will happen on the backend.

**Comment by robhudson** [comment]
Does this also update the URL so we get permalinks?

File: distributionviewer/core/static/js/app/components/views/chart.js (hunk 1 above)
Comment (openjck) [reply distributionviewer/core/static/js/app/components/views/chart.js]: Ah, good catch. I'll revert that. Thanks!

**Comment by openjck** [comment]
> Does this also update the URL so we get permalinks?

Good question. I was thinking we would have the homepage always show the latest charts, and then provide #85 for users who want permalinks to particular dates if that makes sense.

File: distributionviewer/core/static/js/app/components/views/chart.js (hunk 1 above)
Comment (openjck) [reply distributionviewer/core/static/js/app/components/views/chart.js]: Updated!

**Comment by robhudson** [comment]
This looks good to me.

**Comment by spasovski** [comment]
r+ thanks!

**Pull request merged**